### DISCUSSION TITLE
cant update on pc browser

### DISCUSSION THREAD

**Discussion by hardlevel** [darthcloud/BlueRetro#560] [Q&A]
hi!

I just updated my blueretro for 1.8.2, everything seens fine, used HW2 and universal.bin, also tryed with playstation.bin

but after the update, my dualsense connects but any buttons work

can you help?

**Comment by darthcloud** [comment]
Use HW1 firmware

**Comment by hardlevel** [comment]
Hi! it worked with HW1, thanks!

Now I will send data for some popular controllers to you, like apex 3

Em qua., 18 de jan. de 2023 às 09:19, Jacques Gagnon <
***@***.***> escreveu:
